CONTRIBUTING.md

```diff
@@ -63,6 +63,9 @@ The logic for the issue tracker is this -
 
 Anything labeled ["status: good-first-bug"](https://github.com/webcompat/webcompat.com/labels/status:%20good%20first%20bug) is perfect for getting started!
 
+> Note: We do not recommend working on more than two `good-first-bugs`. Take it slow and your time to get into the projects own flow. 
+
```

Comment:
Semantically this is not a blockquote (aka not a citation from somewhere else but I guess we already butchered this elsewhere.) Suggestion:

I feel the sentence is a bit complicated for non English natives:

> Take it slow and your time to get into the projects own flow.

Take your time. Small steps will help you have a better understanding of the project.